ledger/participant-integration-api/src/main/scala/platform/index/ReadOnlySqlLedger.scala

```diff
@@ -55,22 +50,33 @@ private[platform] object ReadOnlySqlLedger {
       maxContractKeyStateCacheSize: Long,
       enableMutableContractStateCache: Boolean,
   )(implicit mat: Materializer, loggingContext: LoggingContext)
-      extends ResourceOwner[ReadOnlyLedger] {
+      extends ResourceOwner[ReadOnlySqlLedger] {
+
+    private val logger = ContextualizedLogger.get(this.getClass)
```

Comment:
Is this intentional to scope the logger to the owner class?

Reply:
Yes. I think it makes more sense like this, since the context provider is actually the `Owner`, right?

Reply:
This means we will get the logger named in the logfile as:
`com.daml.platform.index.ReadOnlySqlLedger.Owner` which is a bit weird

Reply:
I've already moved it back, @mziolekda . Thanks